### PR TITLE
fix: config

### PR DESCRIPTION
**Pull request by devhaozi**
Closes # 

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [x] My code requires changes to the documentation
- [x] I have updated the documentation as required
- [x] All the tests have passed

### CODE REVIEW

File: config/http.go [hunk 1]
```diff
@@ -20,6 +20,10 @@ func init() {
 				"route": func() (route.Route, error) {
 					return ginfacades.Route("gin"), nil
 				},
+				// Optional, default is http/template
+			        "template": func() (render.HTMLRender, error) {
+			            return gin.DefaultTemplate()
```
Comment (hwbrzzl) [inline config/http.go:25]: `gin` doesn't be imported?

**Review by hwbrzzl — COMMENTED** [review]
Good catch!

File: config/http.go (hunk 1 above)
Comment (devhaozi) [reply config/http.go:27]: > `gin` doesn't be imported?

fixed

**Review by hwbrzzl — APPROVED** [review]
LGTM

**Pull request merged**